### DISCUSSION TITLE
missing data van mijn 2e molen.

### DISCUSSION THREAD

**Discussion by BenSpruyt** [jobvk/Home-Assistant-Windcentrale#77] [Support] [answered]
Ik heb aandelen in 2 molens. De Grote Geert en De Trouwe Wachter.
De data van de Grote Geert is compleet. Van de Trouwe Wachter staan een aantal gegevens op nul. 
Alles gereset, maar geen verandering. 
Bijgaand de entities van beide molens naast elkaar.
![Schermafbeelding 2023-02-10 165915](https://user-images.githubusercontent.com/28645268/218138167-9ffb5cd3-925f-4109-9bc3-d78d3d476583.jpg)

**Comment by jobvk** [comment, marked as answer]
De Trouwe wachter en Rode hert hebben inderdaad dit probleem.

Ik kan hier helaas niks aan veranderen. Ik heb de API's gecheckt die de data ophalen. Maar er wordt niks meegegeven vanuit De Windcentrale meegegeven.

Het beste advies wat ik kan geven is contact opnemen met De Windcentrale en kijken of zij er iets aan kunnen doen. Dan zal je nog even moeten wachten of ze het ook gaan repareren. 
https://github.com/jobvk/Home-Assistant-Windcentrale/issues/72#issuecomment-1424643785

**Comment by BenSpruyt** [comment]
Beste Job,

Dank voor het snelle antwoord. Ikzelf mij tot de Windcentrale wenden.
Wie weet!

Groetjes, Ben
